### ISSUE TITLE
Url shorteners implementation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#34]
```
Hi.
I've just created two classes which implement url shortening using is.gd
and 4url.cc services.
I would like to discuss how to add this to the project.
Right now I am just giving the classes code. These have been tested using
both valid and invalid urls.

Feel free to contact me, my e-mail is being included in the files.


```

Original issue reported on code.google.com by `zoe...@gmail.com` on 9 Jan 2010 at 2:48

Attachments:
- [for_url_shorter.vala](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-34/comment-0/for_url_shorter.vala)
- [is_gd_shorter.vala](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-34/comment-0/is_gd_shorter.vala)

**Comment by GoogleCodeExporter** [comment]
```
Hi.
I plan to use bit.ly for url shortening, with custom api keys like in 
http://code.google.com/p/pino-twitter/issues/detail?id=20 and 
http://code.google.com/p/pino-twitter/issues/detail?id=19 issues.
Anyway, thank you for your code.
```

Original comment by `tro...@gmail.com` on 9 Jan 2010 at 3:26

**Comment by GoogleCodeExporter** [comment]
```
So you're only going to support bit.ly?
```

Original comment by `zoe...@gmail.com` on 9 Jan 2010 at 3:32

**Comment by GoogleCodeExporter** [comment]
```
Yep.
```

Original comment by `tro...@gmail.com` on 9 Jan 2010 at 4:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 12 Jan 2010 at 11:03
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 234a21b85a revision. I added goo.gl and is.gd.
```

Original comment by `tro...@gmail.com` on 7 Feb 2010 at 1:59
- Changed state: **Fixed**
